test/compilable/extra-files/ddoc17697.html

```diff
@@ -1,484 +1,506 @@
-
```

Comment:
Err, sorry, this is not about any particular new line, but about the newline characters in the file. If you look at the diff, you'll see that apparently every single line in the file changed - a symptom of the line endings changing (from CR/LF to LF or the other way).